ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-ssh.j2

```diff
@@ -51,9 +51,18 @@ management ssh
 {%     if management_ssh.mac is arista.avd.defined %}
    mac {{ management_ssh.mac | join(" ") }}
 {%     endif %}
+{%     if management_ssh.fips_restrictions is arista.avd.defined(true) %}
```

Comment:
Please adjust the order. From my test on EOS:
```eos
management ssh
   ip access-group ACL-SSH-VRF vrf mgt in
   ip access-group ACL-SSH in
   idle-timeout 15
   connection per-host 10
   fips restrictions
   hostkey client strict-checking
   client-alive interval 666
   client-alive count-max 42
   log-level debug
```

Reply:
For me it is showing below `key-exchange` and `hostkey server`.
 `client-alive` and `hostkey server` position is kept as earlier in the template.

I should move it after `hostkey server` or before `client-alive`?

```
management ssh
   key-exchange ecdh-sha2-nistp521
   hostkey server dsa
   fips restrictions
   hostkey client strict-checking
   client-alive count-max 10
```

Reply:
I think the sequence like:
1. fips restrictions
2. hostkey client strict-checking
3. hostkey server cert sshkey.cert

Right now you have hostkey server before fibs

Reply:
<img width="599" alt="Screenshot 2024-03-01 at 1 11 23 PM" src="https://github.com/aristanetworks/avd/assets/145646625/26d4109d-b7b2-495f-badd-ca38dfd16ac8">
For me it is different, Is this a version difference?

Reply:
Mine is - 4.30.2F-33092737.4302F

Reply:
Moved server_cert below client, as per the CLI